conda-forge.yml

```diff
@@ -1,6 +1,8 @@
+bot:
+  inspection: update-grayskull
```

Comment:
Letting the bot try to update with grayskull is usually helpful and quite a time savings.  I hope you don't mind me turinging this on.